hoomd/md/TwoStepBDGPU.cu

```diff
@@ -343,12 +363,22 @@ hipError_t gpu_brownian_step_one(Scalar4* d_pos,
         dim3 grid((nwork / run_block_size) + 1, 1, 1);
         dim3 threads(run_block_size, 1, 1);
 
+        size_t shared_bytes = (unsigned int)(sizeof(Scalar) * langevin_args.n_types
```

Comment:
```suggestion
       const auto shared_bytes = static_cast<size_t>(sizeof(Scalar) * langevin_args.n_types
```
Might as well convert to the final type immediately. I used auto since the right side mentions type explicitly.